go/obscuronode/nodecommon/rpc/generated/enclave.proto

```diff
@@ -212,8 +212,9 @@ message GetBalanceResponse {
 
 message AttestationReportMsg {
   bytes Report = 1; // The actual report bytes so it can be shared and verified by other nodes
-  bytes PubKey = 2; // public key to encrypt traffic back to this enclave (PKCS1 encoded)
+  bytes PubKey = 2; // Public key to encrypt traffic back to this enclave (PKCS1 encoded)
```

Comment:
Could you get rid of PCKS1 encoded while you're there please - think that was the old RSA encoding method

Reply:
Done.